docs/docs/Deployment/deployment-kubernetes.md

```
@@ -229,20 +229,20 @@ If you want to download the flow from a remote location, you can specify the URL
 
 	```shell
 	helm install my-langinfra-app langinfra/langinfra-runtime -n langinfra --create-namespace --set image.repository=myuser/langinfra-just-chat --set image.tag=1.0.0
-	
+
 	```
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)